server/models/admin.sql

```diff
@@ -265,6 +265,7 @@ END$$
 
  Removes all data from the cost_center_aggregate table and rebuilds it.
 */
+DROP PROCEDURE IF EXISTS zRecalculateCostCenterAggregates$$
```

Comment:
Sorry, my mistake.  This addition was not necessary.  This statement is already in the SQL files somewhere.  When you create the migration file, you can find it.